src/packages/migrate/src/types.ts

```diff
@@ -91,6 +91,7 @@ export namespace EngineResults {
     warnings: any[]
     errors: any[]
     generalErrors: any[]
+    unexecutableMigrations: any[]
```

Comment:
The current schema would be something like:

```ts
interface UnexecutableMigration {
  description: string
}
```

But since we're working on migrate it could change soon-ish.

Reply:
Yes I want to get rid of the any at some point but maybe later then!

Reply:
Just did it